rust/automerge/src/read.rs

```diff
@@ -38,28 +46,34 @@ pub trait ReadDoc {
     /// * If the object ID `obj` is not in the document
     fn path_to_object<O: AsRef<ExId>>(&self, obj: O) -> Result<Vec<(ExId, Prop)>, AutomergeError>;
 
+    fn path_to_object_at<O: AsRef<ExId>>(
```

Comment:
Needs docs. In particular we should clarify what the behavior is here if the object is in the document but was not visible at `heads`